xds/internal/xdsclient/transport/transport.go

```diff
@@ -268,7 +271,11 @@ func (t *Transport) sendAggregatedDiscoveryServiceRequest(stream adsStream, reso
 	if err := stream.Send(req); err != nil {
 		return fmt.Errorf("sending ADS request %s failed: %v", pretty.ToJSON(req), err)
```

Comment:
This is outside of your diff. However, i feel like having the error message after the pretty printed JSON looks weird in the logs. This ErrorF is used to print a longer Warning log in which already has `Sending ADS request for resource` in the message there. 

Can we just return err here and use the message in the actual log statement?

Reply:
This is an example of the entire log 

```
ADS request for {resources: ["xdsclient-test-lds-resourceA"], url: type.googleapis.com/envoy.config.listener.v3.Listener, version: "1", nonce: ""} failed: sending ADS request {
              "versionInfo": "1",
              "node": {
                "id": "fc398f80-0d2d-4560-a03c-7760fa9bcc68"
              },
              "resourceNames": [
                "xdsclient-test-lds-resourceA"
              ],
              "typeUrl": "type.googleapis.com/envoy.config.listener.v3.Listener"
            } failed: EOF  (t=+502.016041ms)

```

Reply:
Done. Changed it to simply return the error here, and let the caller handle the logging. Thanks.